sos/python/Sos.pyx

```diff
@@ -903,7 +903,7 @@ cdef class Schema(SosObject):
         if type(attr_id) == int:
             return Attr(self, attr_id=attr_id)
         elif type(attr_id) == str:
-            return Attr(self, attr_name=attr_id.encode())
+            return Attr(self, attr_name=attr_id)
```

Comment:
@tom95858 because cython option specified `language_level=2`, the `str` inside Sos.pyx is not Python3 string, hence no `encode()` method. I think Python2-str is converted to bytes automatically in Cython-Python3. Without this line change, `ldms-test/store_app_test` failed because `ldms-test/sos_query.py` failed. With this line change, `ldms-test/store_app_test` succeed.